VOCABULARY-CREATION-GUIDE.md

```diff
@@ -2,4 +2,33 @@
 
 Thing Description Forms can be extended with annotations on how to activate affordances.
 These annotations are typically from protocol vocabularies that need to exist separately from the binding template document.
-This document gives guidance how such vocabularies can be created and how they should be used together with the templates and tools provided.
\ No newline at end of file
+This document gives guidance how such vocabularies can be created and how they should be used together with the templates and tools provided.
```

Comment:
This document gives guidance on ...
which tools?
I think the content of the document does not match this sentence, it's a bit misleading. It does not describe how the vocabularies are actually created, rather once we have defined an additional protocol binding vocabulary how to generate the HTML specification.

Reply:
I now tried to add an initial, very rough description regarding the creation of a vocabulary in 2dbb28281679fc48a65d3ba87ac49d436c21862f. If that points in the right direction then we could potentially expand it further – looking forward to your feedback here :)